NEWS.md

```diff
@@ -18,6 +18,7 @@
   to specify the maximum number of visits allowed in non-interactive mode, or asked in interactive
   sessions.
 - Add `predict` method to obtain conditional mean estimates and prediction intervals, both with or without previous observations.
+- Use automatic differentiation to calculate Satterthwaite adjusted degrees of freedom calculation.
```

Comment:
I would add here the advantage from this (speed up etc)

Reply:
added